.gitignore

```diff
@@ -101,4 +101,4 @@ dist
 .dynamodb/
 
 # TernJS port file
-.tern-port
+.tern-port
```

Comment:
If you know how to remove this change, please do. Otherwise, just leave it.

Reply:
No I don't sorry.